addons/linecompass/XEH_preInit.sqf

```diff
@@ -32,7 +32,7 @@ if (hasInterface) then {
             [_key, _pos, GVAR(ACEFingeringColor)] call FUNC(addLineMarker);
 
             [{
-                _this call FUNC(removeLineMarker);
+                call FUNC(removeLineMarker);
```

Comment:
tbh, it may be "officially" supported, but I dislike this as it is not really more readable and less maintainable. Also, didn't the hemtt do this now in the optimizer?